### PR TITLE
AD-1124 feedback/bug fixes

### PR DESCRIPTION
**Pull request by mvcninja**
Bug fixes and updates from feedback

### CODE REVIEW

File: packages/core/package.nls.json [hunk 1]
```diff
@@ -268,7 +268,7 @@
     "AWS.amazonq.toggleCodeSuggestion": "Toggle Auto-Suggestions",
     "AWS.amazonq.toggleCodeScan": "Toggle Auto-Scans",
     "AWS.command.docdb.createCluster": "Create DocumentDB Cluster",
-    "AWS.command.docdb.createInstance": "Create Instance",
+    "AWS.command.docdb.createInstance": "Add Instance...",
```
Comment (kmcginnes) [inline packages/core/package.nls.json:271]: I don't understand the importance of the ellipsis? Typically, that signifies a long running task. But we have only added the ellipsis to two of these actions that I can see in this diff, and I bet there are more than two long running tasks. So does it mean something else or should we be more consistent?

**Review by kmcginnes — APPROVED** [review]

File: packages/core/src/docdb/commands/addRegion.ts [hunk 2]
```diff
@@ -26,6 +27,11 @@ export async function addRegion(node: DBClusterNode | DBGlobalClusterNode): Prom
     return telemetry.docdb_addRegion.run(async () => {
         let globalClusterName = undefined
 
+        if (node.cluster.StorageEncrypted) {
+            void vscode.window.showErrorMessage('Currently supported for unencrypted clusters only.')
```
Comment (alexey-temnikov) [inline packages/core/src/docdb/commands/addRegion.ts:31]: nit: Shall we advise the user to navigate to the AWS console to add a region for Encrypted clusters?

File: packages/core/package.nls.json (hunk 1 above)
Comment (mvcninja) [reply packages/core/package.nls.json:271]: I've added the ellipsis based on 2 factors:
1. Consistency with other parts of the toolkit
2. When the command kicks off a wizard

File: packages/core/src/docdb/commands/addRegion.ts (hunk 2 above)
Comment (mvcninja) [reply packages/core/src/docdb/commands/addRegion.ts:31]: I've removed the menu for this so it's no longer available, but left in the code.  We can add a link to the message if desired.

**Pull request merged**